src/components/remotecontrol/remotecontrol.css

```
@@ -102,7 +134,67 @@
     -ms-user-select: none;
 }
 
-@media all and (orientation: portrait) and (max-width: 50em) {
+.contextMenuList {
+    padding: 1.5em 0;
+}
+
+.contextMenuList i.listItemIcon {
+    font-size: x-large;
+}
+
+.nowPlayingPageImageContainer {
+    width: 16%;
+}
+
+.nowPlayingPageImage {
+    margin: 0 auto;
+    width: 100%;
+}
+
+.nowPlayingSecondaryButtons {
+    display: -webkit-box;
+    display: -webkit-flex;
+    display: flex;
+    -webkit-box-align: center;
+    -webkit-align-items: center;
+    align-items: center;
+    -webkit-flex-wrap: wrap;
+    flex-wrap: wrap;
+    -webkit-box-pack: end;
+    -webkit-justify-content: flex-end;
+    justify-content: flex-end;
```

```suggestion
    justify-content: flex-end;
    z-index: 0;
```

@samuel9554 Possible solution for https://matrix.to/#/!flDUDlxzYhNVyLEcSA:bonifacelabs.ca/$cn9huB5UkI74YUxh_kPblq2v9d7RahiDohsX7Tni-lk?via=plsno.com&via=matrix.org&via=matrix.possumlodge.me